OpenSim/Simulation/Model/Geometry.cpp

```diff
@@ -274,8 +274,9 @@ void Mesh::extendFinalizeFromProperties() {
         try {
             std::ifstream objFile;
             objFile.open(attempts.back().c_str());
-            pmesh.loadFile(attempts.back().c_str());
             // objFile closes when destructed
+            // if the file can be opened but had bad contents e.g. binary vtp 
```

Comment:
We should probably update the exception message that the file could not be found and opened since the mesh won't be read in until display time.